lte/gateway/python/magma/mobilityd/uplink_gw.py

```diff
@@ -60,6 +60,10 @@ def mac_exists(self) -> str:
         return DHCP_Router_Mac_key in self._backing_map
 
     def update_mac(self, value: str):
+        # TODO: enhance check for MAC address sanity.
+        if value.find(':') == -1:
+            return
+
```

Comment:
check with Nick as he may already have a function that checks MAC address sanity.

Reply:
I have plan to use netifaces mac object as TODO.